src/cli.mjs

```diff
@@ -120,46 +120,58 @@ function buildYargs(argvInput) {
         type: "boolean",
         // https://github.com/chalk/chalk?tab=readme-ov-file#chalklevel
         default: chalk.level > 0,
+        group: "Output:",
       },
       config: {
         type: "string",
-        description: "Path to a CLI config file to use. Use `--profile` to select a profile from the file.",
+        description:
+          "Path to a CLI config file to use. Use `--profile` to select a profile from the file.",
         default: ".",
+        group: "Config:",
       },
       profile: {
         alias: "p",
         type: "string",
         description:
           "Profile from the CLI config file to use. Each profile specifies a set of CLI settings.",
         default: "default",
+        group: "Config:",
       },
       user: {
         alias: "u",
         type: "string",
-        description: "User account used to run the command. Register a user account in the CLI using `fauna login`.",
+        description:
+          "User account used to run the command. Register a user account in the CLI using `fauna login`.",
         default: "default",
+        group: "API:",
       },
       json: {
         type: "boolean",
         description: "Output the results as JSON.",
         default: false,
+        group: "Output:",
       },
       quiet: {
         type: "boolean",
-        description: "Only output the results of the command. Useful for scripts, CI/CD, and automation workflows.",
+        description:
+          "Only output the results of the command. Useful for scripts, CI/CD, and automation workflows.",
         default: false,
+        group: "Output:",
       },
       verboseComponent: {
         description:
-          "Components to emit diagnostic logs for. Takes precedence over the `--verbosity` flag. Pass components as a comma-separate list, such as `--verboseComponent fetch, error`, or as separate flags, such as `--verboseComponent fetch --verboseComponent error`.",
+          "Components to emit diagnostic logs for. Takes precedence over the `--verbosity` flag. Pass components as a comma-separate list, such as `--verboseComponent fetch,error`, or as separate flags, such as `--verboseComponent fetch --verboseComponent error`.",
```

Comment:
I can't actually make `--verboseComponent fetch,error` work btw. A problem for later.
https://faunadb.atlassian.net/browse/FE-6176

Reply:
turns out the short syntax is actually `--verboseComponent fetch error argv`. updated!